docs/quickstart.md

```diff
@@ -341,14 +339,14 @@ Here are some things to try for next steps:
 (quickstart-cards)=
 +++
 Check out the following tutorials for more step-by-step guides:
-::::{grid} 2
+::::{grid} 1 1 2 2
 :::{card} Author Enriched MyST Documents ✨
 :link: ./quickstart-myst-documents.md
 Enhance your MyST documents with interactivity, open scholarship, and reproducibility.
 :::
 
 :::{card} Export to static documents 📑
-:link: ./quickstart-myst-documents.md
```

Comment:
@rowanc1 is this change correct? The PR that touched this hasn't deleted that document, it only renamed the _websites_ part (which is correct here).

Reply:
The two links were the same. This changes the second one to point to the static exports!

![image](https://github.com/user-attachments/assets/91ff8a90-f37d-4b05-b28d-2641c99d5352)